packages/cardano-services/src/Program/loadHttpServer.ts

```diff
@@ -14,14 +14,17 @@ import { RabbitMqTxSubmitProvider } from '@cardano-sdk/rabbitmq';
 import { ServiceNames } from './ServiceNames';
 import { TxSubmitHttpService } from '../TxSubmit';
 import { createDbSyncMetadataService } from '../Metadata';
+import { getPool } from './utils';
 import { ogmiosTxSubmitProvider, urlToConnectionConfig } from '@cardano-sdk/ogmios';
 import Logger, { createLogger } from 'bunyan';
 import pg from 'pg';
 
-export interface HttpServerOptions extends CommonProgramOptions {
+export interface HttpServerOptions extends CommonProgramOptions, SrvProgramOptions {
   dbConnectionString?: string;
   dbQueriesCacheTtl: number;
   dbPollInterval: number;
+  retryBackoffFactor: number;
```

Comment:
```suggestion
  serviceDiscoveryBackoffFactor: number;
```
It's a little verbose, but adds the needed context

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/309/commits/0a73034d6d30a95a36abf1fb7a2d069b2ce3cf1e